### PR TITLE
DietPi-Software | Navidrome

### PR DESCRIPTION
**Pull request by Joulinar**
DietPi-Software | Navidrome - add new software title

### CODE REVIEW

**Review by StephanStS — CHANGES_REQUESTED** [review]
Made some changes/additions with my push.

File: docs/software/media.md [hunk 1]
```diff
@@ -1648,4 +1649,87 @@ The Snapcast client will prompt you for the server's IP and port when installing
 
 Source code: <https://github.com/badaix/snapcast>
 
+## Navidrome {: #navidrome }
```
Comment (MichaIng) [inline docs/software/media.md:1652]: Das wird automatisch gesetzt:
```suggestion
## Navidrome
```

File: docs/software/media.md [hunk 2]
```diff
@@ -1648,4 +1649,87 @@ The Snapcast client will prompt you for the server's IP and port when installing
 
 Source code: <https://github.com/badaix/snapcast>
 
+## Navidrome
+
+An open source web-based music collection server and streamer, written in Go.  
+It is API compatible to the [Subsonic media player](http://www.subsonic.org/pages/index.jsp).
```
Comment (ravenclaw900) [inline docs/software/media.md:1655]:
```suggestion
It is compatible with the [Subsonic media player](http://www.subsonic.org/pages/index.jsp).
```

File: docs/software/media.md [hunk 3]
```diff
@@ -1648,4 +1649,87 @@ The Snapcast client will prompt you for the server's IP and port when installing
...
+
+=== "Web interface"
+
+    The web interface uses port **4533**:
+
+    - URL: `http://<your.IP>:4533`
+
+=== "Multiple Music Folders"
+
+    Navidrome don't support multiple music folders out of the box. The feature is still being developed.
```
Comment (ravenclaw900) [inline docs/software/media.md:1667]:
```suggestion
    Navidrome doesn't support multiple music folders out of the box. The feature is still being developed.
```

File: docs/software/media.md [hunk 4]
```diff
@@ -1648,4 +1649,87 @@ The Snapcast client will prompt you for the server's IP and port when installing
...
+=== "Transfer music to DietPi"
+
+    Make sure you have one of DietPi's [File Servers](../file_servers/) installed.  
+    Default music directory:
+
+    - `/mnt/dietpi_userdata/Music`, `/Music` from NFS/Samba
+
+=== "Clients for Navidrome"
+
+    Navidrome has a Subsonic-API compatible server built in, so there should be plenty of clients for different platforms available. See:
```
Comment (ravenclaw900) [inline docs/software/media.md:1724]:
```suggestion
    Navidrome has a Subsonic-compatible server built in, so there should be plenty of clients for different platforms available. See:
```

**Review by ravenclaw900 — COMMENTED** [review]
I've replaced where it says `API compatible` with just `compatible`, however if you want to keep the API part, you can.

**Comment by ravenclaw900** [comment]
Whoops, I commented too late.

**Comment by Joulinar** [comment]
we will add the changes to the release note PR

**Comment by StephanStS** [comment]
@ravenclaw900: I will takeover your suggestions within the PR for the Releasenotes v8.7. Thx.

File: docs/software/media.md (hunk 2 above)
Comment (StephanStS) [reply docs/software/media.md:1655]: Take over this change in the Releasenotes PR.

File: docs/software/media.md (hunk 3 above)
Comment (StephanStS) [reply docs/software/media.md:1667]: Take over this change in the Releasenotes PR.

File: docs/software/media.md (hunk 4 above)
Comment (StephanStS) [reply docs/software/media.md:1724]: Take over this change in the Releasenotes PR.

**Pull request merged**